spec/order_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'order'
+
+describe Order do
+  it "responds to #show_menu" do
+    expect(subject).to respond_to :show_menu
+  end
+
+  it "#show_menu prints the menu" do
+    expect(subject.show_menu).to be_instance_of Array
+  end
+
+  it "responds to #add_pizza with a menu number" do
+    expect(subject).to respond_to(:select_pizza).with(2).arguments
+  end
+
+  it "responds to #selection" do
+    expect(subject).to respond_to :selection
+  end
+
+  it '#selection is an array containing the order' do
+    expect(subject.selection).to be_instance_of Array
+  end
+
+  it '#selection contains the order' do
+    subject.select_pizza(1, 2)
+    expect(subject.selection).to eq [{"1": "Garlic Bread with cheese", price: 6}, {"1": "Garlic Bread with cheese", price: 6}]
+  end
+
+  it "responds to #calculate_total" do
+    expect(subject).to respond_to :calculate_total
+  end
+
+  it "gives the total price of the order" do
+    subject.select_pizza(1, 2)
+    subject.select_pizza(2, 1)
+    expect(subject.calculate_total).to eq 24
+  end
+
+  it "responds to #complete_order" do
+    expect(subject).to respond_to :complete_order
+  end
+
+  it "responds to #send text" do
+    expect(subject).to respond_to :send_sms
+  end
+
+  it "sends a text" do
+    allow(subject).to receive(:send_sms).and_return("You have succesfully ordered your pizza! Total: £#{@total}")
+    expect(subject.complete_order).to eq "You have succesfully ordered your pizza! Total: £#{@total}"
+  end
+end
```

Comment:
As you mentioned earlier putting these alongside 'context' or 'describe' blocks would help readability

Reply:
👍 no Vacuous" tests - all testing real behaviour